jenkins/ocp-config/deploy/jenkins-master.yml

```diff
@@ -62,6 +62,8 @@ parameters:
   required: true
 - name: REPO_BASE
   required: true
+- name: ODS_CLUSTER_GROUPS
```

Comment:
If I understand it correctly, then the cluster groups are meant for provisioning. If this is correct, then the env variable should not be added here, but be passed as a parameter to the pipeline from the provisioning app.

Reply:
@michaelsauter Could you point in the code where I should make that change?

Reply:
@michaelsauter we are bringing a new aspect of the provisioning process. This change provides a way to setup project permission to admin groups independently whether or not the create project pipeline was triggered by provisioning app. I'm highlighting this just to be sure that we are on the same page. If this is a good idea or not is something that we can discuss. What do you think?

Reply:
OpenShift Project provisioning is done through https://github.com/opendevstack/ods-core/blob/master/create-projects/Jenkinsfile. If we add a new parameter to this pipeline then the thing which triggers the pipeline (right now the prov app) should send the parameter. I don't want to have this environment variable as part of every Jenkins deployment.

Reply:
fair enough, thanks for the feedback! @felipecruz91 we need to move the parameter to the ProvApp!

Reply:
Understood, thank you @michaelsauter. I will coordinate with @stitakis to move it to the prov app